pypeit/core/arc.py

```diff
@@ -54,13 +54,7 @@ def setup_param(spectro_class, msarc_shape, fitstbl, arc_idx,
 
     # Instrument/disperser specific
     disperser = fitstbl["dispname"][arc_idx]
-    try:
-        if(fitstbl['binning_x'][arc_idx]):
-            # for XSHOOTER
-            msgs.warn("Binning is imported from binning_x and binning_y.")
-            binspatial, binspectral = fitstbl['binning_x'][arc_idx], fitstbl['binning_y'][arc_idx]
-    except:
-        binspatial, binspectral = parse.parse_binning(fitstbl['binning'][arc_idx])
+    binspatial, binspectral = parse.parse_binning(fitstbl['binning'][arc_idx])
```

Comment:
This is going to overwrite an edit @EmAstro made for X-shooter...

Reply:
Yeah we are coordinating. He made those changes in the wrong place, but the issue is now being corrected in his branch.